### PR TITLE
Fix download link for alibaba.dingtalk

### PR DESCRIPTION
**Pull request by lychichem**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16507)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
​

**Comment by lychichem** [comment]
> image
> `PackageName` needs to be changed to `DingTalk`
> `Publisher` needs to be changed to `Alibaba (China) Network Technology Co., Ltd.`
> `PackageVersion` needs to be changed to `6.0.15-Release.5171263`
> 
> Other than the changes I've made, everything else looks good to me:
> image

Agree. Accept your suggestions. But another question, for Chinese users, should I use `钉钉` as package name?
image

File: manifests/a/alibaba/dingtalk/6.0.15-Release.5171263/alibaba.dingtalk.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 PackageIdentifier: "alibaba.dingtalk"
-PackageVersion: "6.0.15.5171263"
+PackageVersion: "6.0.15-Release.5171263"
 PackageLocale: "zh-CN"
-Publisher: "alibaba"
-PackageName: "dingtalk"
+Publisher: "Alibaba (China) Network Technology Co., Ltd."
```
Comment (ItzLevvie) [inline manifests/a/alibaba/dingtalk/6.0.15-Release.5171263/alibaba.dingtalk.yaml:4]:
```suggestion
Publisher: "Alibaba (China) Network Technology Co.,Ltd."
```
Oops sorry, there isn't supposed to be a space there actually.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
​

**Comment by ItzLevvie** [comment]
> Agree. Accept your suggestions. But another question, for Chinese users, should I use `钉钉` as package name?

You could create a `alibaba.dingtalk.locale.zh-CN.yaml` file.

**Comment by ItzLevvie** [comment]
I updated my comment above: https://github.com/microsoft/winget-pkgs/pull/16507#issuecomment-856019529

**Review by ItzLevvie — APPROVED** [review]

**Comment by lychichem** [comment]
> > Agree. Accept your suggestions. But another question, for Chinese users, should I use `钉钉` as package name?
> 
> You could create a `alibaba.dingtalk.locale.zh-CN.yaml` file for that but first you'll need to convert it to a multi-file manifest (i.e. `alibaba.dingtalk.installer.yaml`, `alibaba.dingtalk.locale.en-US.yaml`, etc.)

I updated the version of alibaba.dingtalk in #17080 , but I am unable to convert it to multi-file manifest. winget fails to validate every time. So I keep it as a single file manifest. Could you please help me?

**Pull request merged**